### PR TITLE
sidemenu

### LINKED ISSUE #1: Side menu
The same as it is in the current app:
- Home
- Schedule
- Workshops
- ngGirls
- Speakers
- Info
- About

### PR DESCRIPTION
**Pull request by Eyzo280**
Resolves #1

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,117 +1,71 @@
...
-      ),
-      home: MyHomePage(title: 'Flutter Demo Home Page'),
+    return Consumer<ThemeNotifier>(
+      builder: (context, theme, child) {
+        return MaterialApp(
+          title: 'ngPolandConf',
+          theme: ThemeData(
+            brightness: Brightness.light,
+            accentColor: Color.fromRGBO(231, 1, 94, 1),
+            primaryColor: Color.fromRGBO(0, 90, 135, 1),
```
Comment (sebastiandenis) [inline lib/main.dart:21]: Change primary color to 59,29,130

File: lib/main.dart [hunk 2]
```diff
@@ -1,117 +1,71 @@
...
-        visualDensity: VisualDensity.adaptivePlatformDensity,
-      ),
-      home: MyHomePage(title: 'Flutter Demo Home Page'),
+    return Consumer<ThemeNotifier>(
+      builder: (context, theme, child) {
+        return MaterialApp(
+          title: 'ngPolandConf',
+          theme: ThemeData(
+            brightness: Brightness.light,
+            accentColor: Color.fromRGBO(231, 1, 94, 1),
```
Comment (sebastiandenis) [inline lib/main.dart:20]: Change accent color to: 255, 0, 122

File: lib/main.dart (hunk 1 above)
Comment (Eyzo280) [reply lib/main.dart]: Changed

File: lib/main.dart (hunk 2 above)
Comment (Eyzo280) [reply lib/main.dart]: Changed

**Pull request merged**